flasksite/queries.py

```diff
@@ -85,4 +85,7 @@
 
 kpiTextInsert = "insert into kpi_content (timestamp, all_text, box_id) values ('%s','%s', '%s')"
 
+# campaign metadata endpoint query
+campaignDataEnpoint_basic_campaign_metadata = "select campaign, sign_ups, new_members, report_backs, all_traffic, start_date, end_date  from overall.overall where nid = '%s'"
```

Comment:
not sure what kind of santizing python does by default, but make sure you can't xss attack this by entering something malicious in the query like
`<script type='text/javascript'>alert('ohsnap');</script>`